src/components/AvatarWithIndicator.tsx

```diff
@@ -11,7 +11,10 @@ import Tooltip from './Tooltip';
 
 type AvatarWithIndicatorProps = {
     /** URL for the avatar */
-    source: UserUtils.AvatarSource;
+    source?: UserUtils.AvatarSource;
+
+    /** account id if it's user avatar */
+    accountID?: number;
```

Comment:
```suggestion
    accountID: number | undefined;
```

This should help catch cases where we are missing to pass accountID.

(do same with Avatar component)

Reply:
Will do it for every Avatar.

What do you think about doing the same for `source`?
Now it looks kinda suspicious that I will have:`source?: string` but `accountID: number | undefined` almost as if accountID is more important than source.